doc/source/features/pxc-strict-mode.rst

```diff
@@ -8,6 +8,10 @@ PXC Strict Mode is designed to avoid the use of
 experimental and unsupported features in |PXC|.
 It performs a number of validations at startup and during runtime.
 
+.. important:: 
+
+   The group replication plugin must be disabled for the strict mode to work correctly.
```

Comment:
Sorry for the late comment, but this isn't entirely correct: strict_mode can be enabled even if the group replication plugin is loaded, if group replication isn't running.

The limiatiton is that group replication can't be running/active in pxc_strict_mode.